.travis_setup.sh

```diff
@@ -2,23 +2,24 @@
 
 set -eux
 
-CHANGED_PWD=false
-# Change the password recreating the root user on mariadb < 10.2
+# Change the password to be empty.
+CHANGED_PASSWORD=false
+# Change the password to be empty, recreating the root user on mariadb < 10.2
 # where ALTER USER is not available.
 # https://stackoverflow.com/questions/56052177/
-CHANGED_PWD_BY_RECREATE=false
+CHANGED_PASSWORD_BY_RECREATE=false
 
 # Install the default used DB if DB is not set.
-if [[ -n ${GITHUB_ACTION-} && -z ${DB-} ]]; then
+if [[ -n ${GITHUB_ACTIONS-} && -z ${DB-} ]]; then
```

Comment:
* I renamed the environment variable `GITHUB_ACTION` to `GITHUB_ACTIONS`. I think the `GITHUB_ACTIONS` showing always `true` on GitHub Actions environment is a right environment variable, and what I wanted to use. See https://docs.github.com/en/actions/reference/environment-variables .